src/Libraries/Microsoft.Extensions.AI.Evaluation.Quality/Microsoft.Extensions.AI.Evaluation.Quality.json

```diff
@@ -190,6 +181,54 @@
         }
       ]
     },
+    {
+      "Type": "sealed class Microsoft.Extensions.AI.Evaluation.Quality.IntentResolutionEvaluator : Microsoft.Extensions.AI.Evaluation.IEvaluator",
+      "Stage": "Experimental",
```

Comment:
The 3 experimental evaluators (IntentResolution,  TaskAdherence, ToolCallAccuracy) were not added to this file because they are still considered in preview.. Can we please revert the corresponding changes?